### PR TITLE
[LUS-118] Patient Care Locations Card

### PR DESCRIPTION
**Pull request by SWKineo**
# Pull Request Process

## Describe the changes you've made to resolve the issue

- Expanded Patient Care page type to include locations
- Added location type
- Added components to the Patient Care page to display a card with each location

### Before
image
image

### After
image
image
image

## Code Review Checklist

- [ ] Have the package.json or package-lock.json been changed? If so, does the pull request description say why?
- [x] Has the UI for mobile and desktop been changed, did you test on IphoneXr and desktop views?
- [x] Does the UI match the requirements in the ticket and designs in figma; does it pass the visual smell test?
- [x] Have TypeScript files been changed, does the TypeScript pass the smell test?
- [ ] If you don't feel super confident in your review, did you assign someone more senior to double check?

### CODE REVIEW

**Comment by SWKineo** [comment]
Adding mobile version in another branch with its own PR

File: app/patient-care/page.tsx [hunk 1]
```diff
@@ -74,7 +58,81 @@ export default async function PatientCare() {
...
+      <LocationsCard locations={patientCarePage.locations} />
       <TabSection fields={patientCarePage.tabSection.fields} />
     </main>
   );
 }
+
+type LocationsCardType = { locations: LocationType[] };
+
+function LocationsCard({ locations }: LocationsCardType) {
+  console.dir(locations);
```
Comment (YolineB) [inline app/patient-care/page.tsx:70]: Remove console command

File: app/patient-care/page.tsx [hunk 2]
```diff
@@ -74,7 +58,81 @@ export default async function PatientCare() {
...
+
+type LocationContentType = { location: LocationType; lastLocation: boolean };
+
+function LocationContent({ location, lastLocation }: LocationContentType) {
+  let outerClassName = "flex flex-col h-full px-7 md:px-6";
+  if (!lastLocation) {
+    outerClassName =
+      outerClassName +
+      " border-b-[1px] md:border-b-0 md:border-r-[3px] border-luskin-green pb-7 mb-7 md:pb-0 md:mb-0 md:mr-1";
+  }
```
Comment (YolineB) [inline app/patient-care/page.tsx:94]: Nice!

File: app/patient-care/page.tsx [hunk 3]
```diff
@@ -74,7 +58,81 @@ export default async function PatientCare() {
 
   return (
     <main>
+      <LocationsCard locations={patientCarePage.locations} />
       <TabSection fields={patientCarePage.tabSection.fields} />
```
Comment (YolineB) [inline app/patient-care/page.tsx:62]: This is particular to tab section but can be placed in this PR - the padding on the tabs sections needs adjustment.

**Review by YolineB — APPROVED** [review]
LGTM!

**Pull request merged**